apps/portal/i18n/locales/it.json

```
@@ -0,0 +1,779 @@
+    "itemsMovedTo": "Elementi spostati in",
+    "downloading": "Download in corso",
+    "video": "Video",
+    "image": "Immagine",
+    "downloaded": "scaricato",
+    "moving": "Spostamento",
+    "toTrash": "nel Cestino...",
+    "movedToTrash": "spostato nel Cestino.",
+    "problemTryAgain": "Si è verificato un problema, riprova",
+    "imageDeleted": "Immagine eliminata con successo",
+    "videoDeleted": "Video eliminato",
+    "noItemsToDelete": "Nessun elemento da eliminare...",
```

Biome (1.9.4)

[error] The key videoDeleted was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)